docs/related-tools/local-analysis/client-side-tools.md

```diff
@@ -88,6 +88,12 @@ The table below describes the supported client-side tools and includes links to
         <td>Faux Pas inspects your iOS or Mac app's Xcode project and warns about possible bugs, as well as about maintainability and style issues.</td>
         <td><a href="https://github.com/codacy/codacy-faux-pas#usage">Running Faux Pas</a> (standalone)</td>
     </tr>
+    <tr>
+        <td>Unity</td>
+        <td><a href="https://github.com/dotnet/roslyn-analyzers">Roslyn Analyzers</a></td>
+        <td><!--TODO Add final description--></td>
```

Comment:
We're missing the tool description (https://github.com/codacy/codacy-roslyn/blob/master/docs/tool-description.md). Please make a first proposal and I'll review it.